src/main.rs

```diff
@@ -1,19 +1,61 @@
 
-#[macro_use] extern crate maplit;
+// btreemap! macro
```

Comment:
this is actually a comment, not commented code

`maplit` enables using the `btreemap!` macro